lemur/tests/test_integration_utility_generate_and_convert.py

```diff
@@ -1,243 +0,0 @@
-# This file consists of testing for functions that generate messages/strings
```

Comment:
I put back the tests that interact with db. Note: some methods like check_existence doesn't have interaction with db so I only keep the copy of its test in the unit_test file.